lib/sorted-set/retrieve-sorted-set.js

```diff
@@ -0,0 +1,21 @@
+const {noopLogger} = require('../utils');
+const { resolveSortedSetValue } = require('./utils');
+const debug = require('debug')('jambonz:realtimedb-helpers');
+
+async function retrieveSortedSet(client, logger, listName) {
```

Comment:
would this be better named 'retrieveFromSortedSet'?

Reply:
fixed